src/App.js

```diff
@@ -1,7 +1,16 @@
 import React from 'react'
+import TimecodeTracker from './components/TimecodeTracker/TimecodeTracker'
+import styled from 'styled-components'
 
-function App() {
-  return <h1>Hello World!</h1>
+export default function App() {
+  return (
+    <div>
```

Comment:
Is the div needed here? Or would <> </> be enough?

Reply:
changed it to h1. thx